docs/user/adding-new-metrics.md

```diff
@@ -1,7 +1,7 @@
 # Adding new metrics
 
-All metrics that your project collects must be defined in a `metrics.yaml` file. 
-This file should be at the root of the application or library module (the same directory as the `build.gradle` file you updated). 
+All metrics that your project collects must be defined in a `metrics.yaml` file.
+This file should be at the root of the application or library module (the same directory as the `build.gradle` file you updated).
```

Comment:
This is Android specific, as there's no `build.gradle` for iOS. I think I'm fine with removing the sentence between the parentheses completely.

Reply:
Ah, I changes this somewhere else. Will change it here.